src/App.js

```diff
@@ -1,26 +1,29 @@
-import React, { Component } from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import { FindMovie } from './components/FindMovie';
+
 import data from './api/movies.json';
 
-export class App extends Component {
-  state = {
-    movies: data,
-  };
+export const App = () => {
+  const [allMovies, setAllMovies] = useState([...data]);
 
-  render() {
-    const { movies } = this.state;
+  const addMovie = (movie) => {
+    const includedMovie = allMovies.find(el => el.imdbId === movie.imdbId);
```

Comment:
```suggestion
    const includedMovie = allMovies.find(el => el.imdbId === movie.imdbId);
```
You could use the method `some`. It's more appropriate for this purpose